README.md

```diff
@@ -13,8 +13,8 @@ source virtualenv/bin/activate
 
 pip install -r requirements.txt
 
-./manage.py loaddata fixtures/templates.json
 ./manage.py migrate
+./manage.py loaddata fixtures/templates.json
```

Comment:
Hvorfor ombytningen ? - fixtures data vil jo hurtigt blive forældede og dermed kræve af blive migreret ?